context/app/static/js/components/Search/SearchBarLayout/SearchBarLayout.jsx

```diff
@@ -1,21 +1,37 @@
+/* eslint-disable max-classes-per-file */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { SearchBox, SortingSelector, ViewSwitcherToggle } from 'searchkit';
+import { SearchBox, SelectedFilters, SortingSelector, ViewSwitcherToggle } from 'searchkit';
 
 import SearchViewSwitch from '../SearchViewSwitch';
 import TilesSortDropdown from '../TilesSortDropdown';
-import { Flex, StyledDiv } from './style';
+import { Flex, StyledDiv, StyledCancelIcon } from './style';
+
+function SelectedFilter(props) {
```

Comment:
Could we pull this out of SearchBarLayout?

Reply:
yep: good call.